### ISSUE TITLE
Unexpected token import

### ISSUE THREAD

**Issue by mmikhan** [vue-bulma/tooltip#7]
I'm using this in a Electron Vue project and unfortunately the tooltip isn't working and shows the following error:

```
Uncaught SyntaxError: Unexpected token import
```
It looks like the console is having an issue with the first line of the `import` but I'm not sure what's could be wrong there. Any thoughts about this?

**Comment by wateryoma** [comment]
i think the issue is about babel-loader config in your project
i ran into same issue, but i solved it by copying tooltip from node_modules to my project src

**Comment by mmikhan** [comment]
I've investigated this a lot but couldn't find any which is even related to the `.babelrc` file. I was using this with the [Electron-Vue](https://github.com/SimulatedGREG/electron-vue) boilerplate.

**Comment by wateryoma** [comment]
I'm using it with the same template, we have the same case.
I just copied all vue bulma components i needed inside renderer components folder - not very elegant but this way everything works

**Comment by mmikhan** [comment]
Yeah, it may work but I don't want to manage any dependencies without the dependency manager. I'd rather switch to a different library or wait until it's fixed.

**Comment by wateryoma** [comment]
ok, let's hope it'll be fixed soon
probably you should report this issue at [Electron-Vue](https://github.com/SimulatedGREG/electron-vue)

**Comment by p418** [comment]
I'm also using `electron-vue` boilerplate and managed to make it work

You need to add  `vue-bulma-*` module to webpack's whitelist .

```js
// .electron-vue/webpack.renderer.config.js:21

let whiteListedModules = ['vue', 'vue-bulma-collapse', 'vue-bulma-tooltip', 'vue-bulma-notification']
let rendererConfig = {
  devtool: '#cheap-module-eval-source-map',
  entry: {
    renderer: path.join(__dirname, '../src/renderer/main.js')
  },
....

```
